### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by ocefpaf**
@asmeurer here are some comments/suggestion on you `aspell` recipe. This build on the `conda-forge` Linux docker image. Hopefully it will be OK on `OS X` too.

### CODE REVIEW

**Comment by asmeurer** [comment]
Why are you using four space indentation? Two spaces is pretty standard for yaml.

File: recipes/aspell-en/build.sh [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#!/bin/bash
```
Comment (asmeurer) [inline recipes/aspell-en/build.sh:1]: Why is this needed? build.sh isn't even executable.

**Comment by ocefpaf** [comment]
> Why are you using four space indentation? Two spaces is pretty standard for yaml.

We don't enforce two nor four in `conda-forge`. That was just how my text editor set. You can close this and copy what you think is necessary if you don't like 4. I am fine either way.

**Comment by asmeurer** [comment]
Well aspell hasn't been updated since 2011, so I'm not holding out. Apparently the dictionary has an update, and it's not even versioned the same way [any more](ftp://ftp.gnu.org/gnu/aspell/dict/0index.html).

File: recipes/aspell-en/build.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
-./configure --vars ASPELL="$PREFIX/bin/aspell"
+#!/bin/bash
+
+bash configure --vars ASPELL="$PREFIX/bin/aspell"
```
Comment (ocefpaf) [inline recipes/aspell-en/build.sh:3]: You'll need at least this part. Although `bash` did not work on `OS X` and you can try  making it an executable and/or calling with `sh`.
